src/Components/ErrorPage/ErrorPage.js

```diff
@@ -1,10 +1,10 @@
 import './ErrorPage.css';
 
-const ErrorPage = () => {
+const ErrorPage = ({ message }) => {
 
   return (
-    <div>
-      Hello from the error page!
+    <div className="error-display">
+      <h4 className="error-message">{message}</h4>
```

Comment:
Just an extra thought, what happens on your undefined path? Like '/asldkjf'? It can be nice UI for that path/message in particular to add a link home. Like "Page not found, would you like to go home?" and then if message is "page not found" add that navlink to '/'!